docs/overview.md

```diff
@@ -183,3 +219,19 @@ Quantum plugin will be using Spank architecture events sequence of call during j
         2. Release resource
 
 ![general architecture](./images/general_architecture.png)
+
+## Architectural Tenents
+
+* A Slurm-level QPU resource maps to physical resource of a Quantum Computer
+  * Quantum backend selection is part of that slurm resource definition and is considered configuration, not source code
+  * Qiskit-level code can refer to that slurm resource and access/use the quantum resource behind it. Qiskit-level code should avoid naming the desired backend directly (=> it should be part of the slurm resource definition instead)
+  * slurm QPU resources have an identity (allowing to bind against it from qiskit)
+  * additional qualifiers of the slurm QPU resource are backend type specific
+  * parallelism abstracts (such as execution lanes which are anonymous units to prepare jobs in parallel for quantum execution which is still serialized) are abstracted behind the slurm QPU resource. Qualifiers may be used to deal with specifics (such as: are these lanes held exclusive for one user, or is there a shared access possible)
+* Quantum resources are acquired/locked before usage
+  * as identification/selection of the quantum resource is through the slurm resource, transpilation can only happen after that
```

Comment:
This will probably change as resource consumption will be crazy with transpilation, but for now we can leave it here.

Reply:
Adding a comment. Agree this feels like MVP0 and needs a more elaborated concept as we move forward